src/components/alert/notes.md

```diff
@@ -0,0 +1,9 @@
+# Designer Notes
+- Useful to perform an action in context without navigating the user to a separate page.
+- Several pre-set widths are available - the height of the dialog will flex to fit the content. It’s best to avoid dialogs that are taller than the user’s viewport height. Typical user viewport heights can be as little as 650 pixels.
+- Choose whether a dark tint is applied behind the dialog which helps to focus the user on the dialog.
+- A configuration shows a close icon at the top right of the Dialog. Sometimes users are more likely to click this than a traditional ‘Cancel’ button.
+
+# Related Components
+- Complex task that needs more space? Try Dialog Full Screen. <!-- TODO link -->
```

Comment:
Are we unable to add the `link`s at this stage?

Reply:
I added it. Thanks.